src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -829,4 +829,17 @@ Valid from:</comment>
     <value>Path: {0}</value>
     <comment>0 - folder path</comment>
   </data>
+  <data name="AnAbsoluteURIIsRequired" xml:space="preserve">
+    <value>An absolute URI is required.</value>
+  </data>
+  <data name="HTTPOrHTTPSIsRequired" xml:space="preserve">
+    <value>HTTP or HTTPS is required.</value>
+  </data>
+  <data name="TimestampServiceRespondedError" xml:space="preserve">
+    <value>The timestamp service responded with HTTP status code '{0}' ('{1}').</value>
+    <comment>{0} is the httpResponse.StatusCode, {1} is the httpResponse.ReasonPhrase.</comment>
+  </data>
+  <data name="TimestampServiceRespondedInvalidFormat" xml:space="preserve">
+    <value>The reply from the time stamp server was in a invalid format.</value>
```

Comment:
The [sentence I provided](https://github.com/NuGet/NuGet.Client/pull/3168#discussion_r363851151) was:
```
The reply from the timestamp server was invalid.
```
*  It's `timestamp` not `time stamp`.
*  `a invalid` is incorrect grammar.

Please use the sentence I provided.

Reply:
Thanks! Fixed.